README.md

```diff
@@ -258,3 +258,24 @@ mock
     )
   );
 ```
+
+## History
+
+The `history` property allows you to enumerate existing axios request objects. The property is an object of verb keys referencing arrays of request objects.
+
+This is useful for testing.
+
+```js
+describe('Feature', () => {
+  it('requests an endpoint', () => {
+    var mock = new AxiosMockAdapter(axios);
+    mock.onGet('/endpoint').replyOnce(200);
+    
+    feature.request();
```

Comment:
@LukeeeeBennett This is probably an asynchronous call so you would either need a Promise / callback / await here. 😃

Reply:
Doh, teaches me for getting too specific with examples. Fixing!

Reply:
@gitlab-winnie How's it looking?

Reply:
Looks good to me! ✨